Services/CO.CDP.Organisation.WebApi/IIdentifierService.cs

```diff
@@ -0,0 +1,6 @@
+namespace CO.CDP.Organisation.WebApi;
+
+public interface IIdentifierService
+{
+    string? GetRegistryUri(string? scheme, string? identifierId);
```

Comment:
Why are `scheme` and `identifierId` nullable?

Reply:
Scheme previously was and not anymore with current usage, so changed that.
IdentifierId seems to be made purposely nullable, even though not sure what value such entry would bring.